masonite_cli/commands/InstallCommand.py

```diff
@@ -11,11 +11,13 @@ class InstallCommand(Command):
     install
         {--no-key : If set, craft install command will not generate and store a new key}
         {--no-dev : If set, Masonite will install with dev dependencies}
+        {--f|--force=False : Overwrite .env if exists}
```

Comment:
This option won't work because it requires a value. What you need is:

```
{--f|--force: Overwrite .env if exists}
```

Reply:
to be honest, I didn't test it, thanks for review, addressed.